pkgs/development/python-modules/dmt-core/default.nix

```
@@ -0,0 +1,52 @@
+{ lib, fetchPypi, buildPythonPackage, python3Packages, colorama, colormath
+, cycler, h5py, joblib, matplotlib, more-itertools, packaging, pandas, pint
+, pyarrow, pyqtgraph, pyyaml, requests, scikit-rf, scipy, semver, setuptools
+, reprint, verilogae, pytest }:
+
+buildPythonPackage rec {
+  pname = "dmt-core";
+  version = "2.1.0";
+  pyproject = true;
+
+  src = fetchPypi {
+    inherit version;
+    pname = "DMT_core";
+    hash = "sha256-489E+uNn4NgyCwxsUMEPH/1ZuM+5uNq4zx8F88rkHMU=";
+  };
+
+  propagatedBuildInputs = [
+    colorama
+    colormath
+    cycler
+    h5py
+    joblib
+    matplotlib
+    more-itertools
+    packaging
+    pandas
+    pint
+    pyarrow
+    pyqtgraph
+    pyyaml
+    requests
+    scikit-rf
+    scipy
+    semver
+    setuptools
+    reprint
+    verilogae
+  ];
+
+  nativeBuildInputs = [ pytest ];
+
+  meta = {
+    changelog =
+      "https://gitlab.com/dmt-development/dmt-core/-/blob/Version_${version}/CHANGELOG?ref_type=tags";
+    description =
+      "Python tool to help modeling engineers extract model parameters, run circuit and TCAD simulations and automate infrastructure.";
```

Has this been produced by the rfc style formatter or is this based on some verbal description of a wanted formatting result? I am only aware of these being single lines per key. =>

```suggestion
    changelog = "https://gitlab.com/dmt-development/dmt-core/-/blob/Version_${version}/CHANGELOG?ref_type=tags";
    description = "Python tool to help modeling engineers extract model parameters, run circuit and TCAD simulations and automate infrastructure.";
```

---

Yes this was the result of `nixfmt`

---

RFC style or classic? Because locally RFC style created them like in my suggestion… It did also split the argset into one entry per line, rather than everything cramped into 3…

---

I wonder if `nixfmt` installs `nixfmt-classic`. I see there's a specific `nixfmt-rfc-style` which I'll have to make use of.